docs:android 7.6.8 release (#21344)
newrelic/docs-website: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-768.mdx

---
subject: Android agent
releaseDate: '2025-07-29'
version: 7.6.8
---

## Bug Fixes
- Addressed a problem preventing stack traces from being displayed correctly for Unity errors.
- Addressed an issue that prevented summary metrics from being recorded when the app was foregrounded after the initial launch.

> - Fixed an issue where errors were thrown due to invalid Log JSON entries.

Reviewer: ```suggestion
- Fixed an issue where invalid JSON in log entries could cause errors.
```
